packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/nodejs/sample-app.md

```diff
@@ -85,11 +87,17 @@ The Event Hook use case is a simple local application response to the addition o
     //userCreated Event request, POST from Okta
 
     app.post("/userCreated", (request, response) => {
+
+        var newUser = request.body.data.events[0].target[0]["alternateId"];
         console.log(" ");
-        console.log('The user ' + request.body.data.events[0].target[0]["alternateId"] +
-        " has been added to the Okta org!");
-        response.sendStatus(200);
+        if (newUser == null){
+            console.log("No user in request! Update the JSON target field with a value.");
+        } else {
+            console.log('The user ' + newUser +
```

Comment:
just a nit here: a mix of single quotes and double quotes are used for strings in this log entry

Reply:
Thanks! Good catch.